### PR TITLE
Home Work 10 Exercise 2 Simple DB

### PR DESCRIPTION
**Pull request by a-vov4eg**

### CODE REVIEW

File: homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package bean;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class Book implements Serializable {
+    private long id;
+    private static long idInc = 1;
```
Comment (TLittleProgrammer) [inline homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java:8]: idInc - ?

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

**Review by vladis1350 — APPROVED** [review]
lgfm

File: homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java (hunk 1 above)
Comment (javaGuruBY) [reply homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java:8]: GUID

Comment (javaGuruBY) [reply homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java:8]: AthomicLong

File: homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    private String name;
+    private String author;
+    private int pagesCount;
+
+    public Book() {
+    }
+
+    public Book(String name, String author, int pagesCount) {
+        this.id = idInc++;
```
Comment (javaGuruBY) [inline homework/Avseenkov V/HomeWork10/Ex2_SimpleDB/bean/Book.java:17]: private setAndIncementId() {
    this.is = Book.idInc;
    Book.idInc++;
}

**Review by javaGuruBY — APPROVED** [review]
lgfm

**Pull request merged**